docs/content/release-notes/flink-2.0.md

```diff
@@ -0,0 +1,1666 @@
+
+##### [FLINK-36287](https://issues.apache.org/jira/browse/FLINK-36287)
+
+When the sink writer and committer are not chained, it's possible that committables become part of the channel state. However, then it's possible that they are not received before notifyCheckpointComplete. Further, the contract of notifyCheckpointComplete dictates that all side effects need to be committed or we fail on notifyCheckpointComplete. This contract is essential to final checkpoints, so exactly-once sinks don't use unaligned checkpoints anymore to send metadata around transactions now.
+
+### Configuration
+
+#### Migrate string configuration key to ConfigOption
+
+##### [FLINK-34079](https://issues.apache.org/jira/browse/FLINK-34079)
+
+We have deprecated all `setXxx` and `getXxx` methods except `getString(String key, String defaultValue)` and `setString(String key, String value)`, such as: `setInteger`, `setLong`, `getInteger` and `getLong` etc. We strongly recommend users and developers use get and set methods directly.
+
+In addition, we recommend users to use ConfigOption instead of string as key.
+
+#### Remove flink-conf.yaml from flink dist
+
+##### [FLINK-33677](https://issues.apache.org/jira/browse/FLINK-33677)
+
+[FLIP-366](https://cwiki.apache.org/confluence/x/YZuzDw) introduces support for parsing standard YAML files for Flink configuration. A new configuration file named config.yaml, which adheres to the standard YAML format, has been introduced.
+
+In the Flink 2.0, we have removed the old configuration file from the flink-dist, along with support for the old configuration parser. Users are now required to use the new config.yaml file when starting the cluster and submitting jobs. They can utilize the migration tool to assist in migrating legacy configuration files to the new format.
+
+### MISC
+
+#### DataStream V2 API
+
+##### [FLINK-34547](https://issues.apache.org/jira/browse/FLINK-34547)
+
+The DataStream API is one of the two main APIs that Flink provides for writing data processing programs. As an API that was introduced practically since day-1 of the project and has been evolved for nearly a decade, we are observing more and more problems of it. Improvements on these problems require significant breaking changes, which makes in-place refactor impractical. Therefore, we propose to introduce a new set of APIs, the DataStream API V2, to gradually replace the original DataStream API.
+
+In Flink 2.0, we provide the MVP version of the new DataStream V2 API. It contains the low-level building blocks (DataStream, ProcessFunction, Partitioning), context and primitives like state, time service, watermark processing. At the same time, we also provide some high-level extensions, such as window and join. They are more like short-cuts / sugars, without which users can probably still achieve the same behavior by working with the fundamental APIs, but would be a lot easier with the builtin supports.
```

Comment:
what do we mean by MVP here. In what sense is it a minimum viable product. Is MVP like a beta - as such not as robust as we would expect. Or would it be better to say  In Flink 2.0, we begin to introduce new DataStream V2 API; expect this to be fleshed out in the 2.x releases.

"They are more like short-cuts / sugars, without which users can probably still achieve the same behavior by working with the fundamental APIs, but would be a lot easier with the builtin supports." I would replace this with something like, The high level extensions make it is simpler to work with the APIs for many cases, though if you want more flexibility / control,  you can use the low level APIs.